proxy/src/main/java/com/wavefront/agent/logsharvesting/EvictingMetricsRegistry.java

```diff
@@ -52,7 +46,10 @@
 
   public Counter getCounter(MetricName metricName, MetricMatcher metricMatcher) {
     metricNamesForMetricMatchers.get(metricMatcher).add(metricName);
-    return (Counter) metricCache.get(metricName, metricsRegistry::newCounter);
+    // use delta counters instead of regular counters. It helps with load balancers present in
+    // front of proxy (PUB-125)
+    return (Counter) metricCache.get(metricName, key -> DeltaCounter.get(metricsRegistry,
```

Comment:
We should add a comment that anyone who uses EvictingMetricsRegistry will end up dealing with delta counters.
Right now only LogsIngester uses EvictingMetricsRegistry, so we should be okay here. But a comment will be useful.

Reply:
Added comment.